src/components/Company/DocumentSigner/DocumentList/Head.tsx

```diff
@@ -1,7 +1,7 @@
 import { useTranslation } from 'react-i18next'
 
 export function Head() {
-  const { t } = useTranslation('Company.DocumentSigner')
+  const { t } = useTranslation('Company.DocumentList')
```

Comment:
No idea what happened with these but there must have been some regression 😅 it has always been named DocumentList. This updates to fix